packages/elements-dev-portal/src/version.ts

```diff
@@ -1,2 +1,2 @@
 // auto-updated during build
-export const appVersion = '1.5.1';
+export const appVersion = '1.6.2';
```

Comment:
What's this?

Reply:
I've seen this get auto-bumped before too, but I have no idea what causes it. 🤔

Reply:
This is autogenerated during release process anyways - so what is in this file in src doesn't really matter.

Used to communicate the version of the library to `elements-dev-portal`.